### PR TITLE
housekeeping: upgrade minimist@^1.2.6

### PR DESCRIPTION
**Pull request by dschaller**
### Description

Upgrade minimist to ^1.2.6 to pick up [CVE patch](https://github.com/advisories/GHSA-xvch-5gv4-984h)

### Testing Performed

manual / ci

### CODE REVIEW

**Review by jdslaugh — APPROVED** [review]

File: frontend/yarn.lock [hunk 1]
```diff
@@ -16058,12 +16058,7 @@ minimist-options@4.1.0:
     is-plain-obj "^1.1.0"
     kind-of "^6.0.3"
 
-minimist@^1.1.1, minimist@^1.1.3, minimist@^1.2.0, minimist@^1.2.5:
-  version "1.2.5"
-  resolved "https://registry.yarnpkg.com/minimist/-/minimist-1.2.5.tgz#67d66014b66a6a8aaa0c083c5fd58df4e4e97602"
-  integrity sha512-FM9nNUYrRBAELZQT3xeZQ7fmMOBg6nWNmJKTcgsJeaLstP/UODVpGsr5OhXhhXg6f+qtJ8uiZ+PUxkDWcgIXLw==
```
Comment (Yiwen-Gao) [inline frontend/yarn.lock:16064]: Why was this deleted?

**Review by Yiwen-Gao — APPROVED** [review]

File: frontend/yarn.lock (hunk 1 above)
Comment (dschaller) [reply frontend/yarn.lock:16064]: Because the matching pattern below has been updated to all resolve to 1.2.6 (e.g. minimist @ 1.1.1, 1.1.3, ^1.2.0, ^1.2.5 all resolve the the version 1.2.6

Comment (Yiwen-Gao) [reply frontend/yarn.lock:16064]: Oh oops, I should've read more closely. Thanks!

Comment (dschaller) [reply frontend/yarn.lock:16064]: no worries, lockfiles are very odd :D
